extension.js

```diff
@@ -332,15 +332,15 @@ const UserWidget = GObject.registerClass(class UserWidget extends St.BoxLayout {
     }
 
     openUserAccount() {
-        Util.spawn(['/bin/bash', '-c', "gnome-control-center user-accounts"]);
+        Util.spawn(["/usr/bin/env bash", '-c', "gnome-control-center user-accounts"]);
```

Comment:
Hi, In my Manjaro is an error `/user/bin/env bash` is not found.  Maybe some condition about program exists should be there. Or add an option to the prefs to set the executable. Default  could be `/bin/bash` but it' might be changed to `"/usr/bin/env bash"`

Reply:
> `/user/bin/env` bash

It should be `/usr/bin/env`

Reply:
hm.. but I've tested in console, that,  `/usr/bin/env bash` starts `bash` hm..

Reply:
`/usr/bin/env bash -c gnome-control-center user-accounts` works perfectly fine in console for me

Reply:
Tofu Menu seems to just omit `bash` altogether and call the command directly: https://github.com/tofutech/tofumenu/blob/1cb92805d7a58971a9a5c287fbd8efd0a56d20c4/extension.js#L16

Reply:
Or even `/bin/sh` would work fine

Reply:
Please check this https://github.com/pawelswiszcz/Avatar-Gnome-Shell-Extension/pull/12/files . In Manjaro it works

Reply:
Yes it works!

Reply:
Thanks!. Realising PR